whylog/config/__init__.py

```diff
@@ -28,6 +35,42 @@ def _filter_rule_set(self, parsers_list):
     def _get_locations_for_logs(self, logs_types_list):
         pass
 
+    def _load_parsers(self):
+        parsers_definitions = self._load_file_with_config(self._parsers_path)
+        log_type_manager = LogTypeManager({})
```

Comment:
Passing empty dictionary to a Manager shouldn't be necessary. Initialize dictionary in `__init__`.